charts/node/values.yaml

```diff
@@ -227,6 +227,16 @@ node:
   #   seed: "//Alice"
   #   extraDerivation: //babe
 
+  ## Inject keys from already existing kubernetes secrets
+  existingSecrets:
+    ## List of keys to be added to keystore
```

Comment:
Can you document the secret format to be used here.
1. State that the value of `node.existingSecrets.keys` should be a list of Kubernetes secret names.
2. Document that the secret should contains the 3 keys: `type`, `scheme` and `seed`.